lang/go/go.py

```diff
@@ -0,0 +1,213 @@
+        actions.insert(" *= ")
+
+    def code_operator_exponent():
+        actions.insert(" ^ ")
+
+    def code_operator_division():
+        actions.insert(" / ")
+
+    def code_operator_division_assignment():
+        actions.insert(" /= ")
+
+    def code_operator_modulo():
+        actions.insert(" % ")
+
+    def code_operator_modulo_assignment():
+        actions.insert(" %= ")
+
+    def code_operator_equal():
+        actions.insert(" == ")
+
+    def code_operator_not_equal():
+        actions.insert(" != ")
+
+    def code_operator_greater_than():
+        actions.insert(" > ")
+
+    def code_operator_greater_than_or_equal_to():
+        actions.insert(" >= ")
+
+    def code_operator_less_than():
+        actions.insert(" < ")
+
+    def code_operator_less_than_or_equal_to():
+        actions.insert(" <= ")
+
+    def code_operator_and():
+        actions.insert(" && ")
+
+    def code_operator_or():
+        actions.insert(" || ")
+
+    def code_operator_bitwise_and():
+        actions.insert(" & ")
+
+    def code_operator_bitwise_and_assignment():
+        actions.insert(" &= ")
+
+    def code_operator_increment():
+        actions.insert("++")
+
+    def code_operator_bitwise_or():
+        actions.insert(" | ")
+
+    def code_operator_bitwise_exclusive_or():
+        actions.insert(" ^ ")
+
+    def code_operator_bitwise_left_shift():
+        actions.insert(" << ")
+
+    def code_operator_bitwise_left_shift_assignment():
+        actions.insert(" <<= ")
+
+    def code_operator_bitwise_right_shift():
+        actions.insert(" >> ")
+
+    def code_operator_bitwise_right_shift_assignment():
+        actions.insert(" >>= ")
+
+    def code_self():
+        actions.insert("this")
+
+    def code_operator_object_accessor():
+        actions.insert(".")
+
+    def code_insert_null():
+        actions.insert("nil")
+
+    def code_insert_is_null():
+        actions.insert(" == nil")
+
+    def code_insert_is_not_null():
+        actions.insert(" != nil")
+
+    def code_state_if():
+        actions.user.insert_between("if ", " ")
+
+    def code_state_else_if():
+        actions.user.insert_between("else if ", " ")
+
+    def code_state_else():
+        actions.insert("else ")
+        actions.key("enter")
+
+    def code_state_switch():
+        actions.user.insert_between("switch (", ") ")
+
+    def code_state_case():
+        actions.user.insert_between("case ", " :")
+
+    def code_state_for():
+        actions.user.insert_between("for ", " ")
+
+    def code_state_while():
+        actions.user.insert_between("for ", " ")
+
+    def code_break():
+        actions.insert("break")
+
+    def code_next():
+        actions.insert("continue")
+
+    def code_insert_true():
+        actions.insert("true")
+
+    def code_insert_false():
+        actions.insert("false")
+
+    def code_import():
+        actions.insert("import ")
+
+    def code_private_function(text: str):
+        actions.insert("private")
```

Comment:
This duplicate of `code_private_function`.  I believe the other one is what we want
```suggestion
```